### PR TITLE
Mongoose UserCrud operation

### PR DESCRIPTION
**Pull request by jitendrasbunde**

### CODE REVIEW

File: mongooseCrud/config.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  url: `mongodb://127.0.0.1:27017/ajay`
```
Comment (sunil4sonawane) [inline mongooseCrud/config.js]: db name should not be in URL

File: mongooseCrud/index.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+}).then(() => {
+    console.log("Successfully connected to the database");    
+}).catch(err => {
+    console.log('Could not connect to the database. Exiting now...', err);
+    process.exit();
+});
+
+// define a simple route
+app.get('/', (req, res) => {
+    res.json({"message": "Welcome to EasyNotes application. Take notes quickly. Organize and keep track of all your notes."});
```
Comment (sunil4sonawane) [inline mongooseCrud/index.js]: This is copy paste line

File: mongooseCrud/mongooseConnection.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+//const dbConfig = require('./config.js');
+const mongoose = require('mongoose');
+mongoose.Promise = global.Promise;
+const username = process.env.username;
+const password = process.env.password;
+const dbName = process.env.dbName;
+
+exports.connect =()=>{
+  mongoose.connect(`mongodb://127.0.0.1:27017/${dbName}?authSource=admin`, {
```
Comment (sunil4sonawane) [inline mongooseCrud/mongooseConnection.js]: url should be fetched from env

**Review by sunil4sonawane — CHANGES_REQUESTED** [review]
Where is login API and JWT auth? Please go through the comments as well

File: mongooseCrud/config.js (hunk 1 above)
Comment (jitendrasbunde) [reply mongooseCrud/config.js]: sir mene url mongooseConnection mw hi likhi he

File: mongooseCrud/index.js (hunk 2 above)
Comment (jitendrasbunde) [reply mongooseCrud/index.js]: change sir

File: mongooseCrud/mongooseConnection.js (hunk 3 above)
Comment (jitendrasbunde) [reply mongooseCrud/mongooseConnection.js]: port no and database name fetch by user

**Review by sunil4sonawane — APPROVED** [review]

**Pull request merged**